src/FSharp.Control.R3/Observable.fs

```diff
@@ -11,13 +12,38 @@ let inline bind ([<InlineIfLambda>] f : 'T -> Observable<'TNext>) source = Obser
 /// Converts the elements of the sequence to the specified type
 let inline cast<'T, 'CastType> (source) = ObservableExtensions.Cast<'T, 'CastType> (source)
 
+let inline catch ([<InlineIfLambda>] f : 'Exn -> Observable<'TNext>) o = ObservableExtensions.Catch (o, f)
```

Comment:
Because the R3 exception handling significantly differs from Rx, this could need some xml comments as well. (R3 docs: "Stopping the pipeline at OnError is a mistake.")

Reply:
Added XML comment with remarks